performance_test/src/experiment_execution/analyze_runner.cpp

```diff
@@ -54,6 +75,35 @@ AnalyzeRunner::AnalyzeRunner()
     m_sub_runners.push_back(DataRunnerFactory::get(m_ec.topic_name(), m_ec.com_mean(),
       RunType::SUBSCRIBER));
   }
+
+#ifdef ODB_FOR_SQL_ENABLED
+#ifdef PERFORMANCE_TEST_ODB_SQLITE
+  m_db = std::unique_ptr<odb::core::database>(new odb::sqlite::database(
+        m_ec.db_name(), SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE));
+#elif PERFORMANCE_TEST_ODB_MYSQL
+  m_db = std::unique_ptr<odb::core::database>(new odb::mysql::database(
+        m_ec.db_user(), m_ec.db_password(), m_ec.db_name(), m_ec.db_host(), m_ec.db_port()));
+#elif PERFORMANCE_TEST_ODB_PGSQL
+  m_db = std::unique_ptr<odb::core::database>(new odb::pgsql::database(
+        m_ec.db_user(), m_ec.db_password(), m_ec.db_name(), m_ec.db_host(), m_ec.db_port()));
+#endif
+  {
+    #ifdef PERFORMANCE_TEST_ODB_SQLITE
+    odb::core::connection_ptr c(m_db->connection());
+    c->execute("PRAGMA foreign_keys=OFF");
```

Comment:
Why do we disable foreign keys?

Reply:
I was getting "foreign  key constraint failed" error that is a known issue (described in 18.5.3. section in manual): and the solution they give: 
```
When  foreign  key  constraints  checking  is  enabled,  then  you  may  get  the 
"foreign  key constraint failed" error while re-creating the database schema. 
This error is due to bugs in theSQLite  DDL  foreign  keys  support. 
The  recommended  work-around  for  this  problem  is  to  temporarily  disable  
foreign  key  constraints  checking  while  re-creating  the  schema.
```

Reply:
Then please comment this in the code.

Reply:
@monidzik: Is this fixed?

Reply:
yes, see here https://github.com/ApexAI/performance_test/pull/84#issuecomment-538944793